src/main/java/github/weichware10/toolbox/Main.java

```diff
@@ -1,86 +1,119 @@
 package github.weichware10.toolbox;
 
-import github.weichware10.toolbox.gui.ConfirmBox;
-import github.weichware10.toolbox.gui.Startbildschirm;
+import github.weichware10.toolbox.gui.App;
+import github.weichware10.toolbox.gui.dialogs.ConfirmDialog;
+import io.github.cdimascio.dotenv.Dotenv;
+import java.io.File;
 import java.io.IOException;
 import java.io.OutputStream;
 import java.io.PrintStream;
 import javafx.application.Application;
-import javafx.event.EventHandler;
-import javafx.scene.Scene;
-import javafx.scene.control.TextArea;
+import javafx.geometry.Rectangle2D;
 import javafx.scene.image.Image;
+import javafx.stage.Screen;
 import javafx.stage.Stage;
 import javafx.stage.WindowEvent;
+import org.joda.time.DateTime;
 
 /**
  * Toolbox GUI.
- *
  */
 public class Main extends Application {
+    /**
+     * Einstiegspunkt der Toolbox - Einführungsbildschirm.
+     *
+     * @param args - args
+     */
     public static void main(String[] args) {
+
+        // in Datei und Konsole loggen
+        String location = Dotenv.configure().directory(".").load().get("LOGS");
+        PrintStream ps = new PrintStream(new Console(location, System.out), true);
+        System.setOut(ps);
+        System.setErr(ps);
+
         launch(args);
     }
 
+    /**
+     * setzt allgemeine Werte der primaryStage und startet die App.
+     * @param primaryStage - das Hauptfenster
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.RequireEmptyLineBeforeBlockTagGroupCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Javadoc tag '@param' should be preceded with an empty line.